packages/@aws-cdk/aws-stepfunctions-tasks/lib/evaluate-expression.ts

```diff
@@ -110,6 +110,22 @@ function createEvalFn(runtime: lambda.Runtime, scope: Construct) {
         lambdaPurpose,
         code: lambda.Code.fromAsset(path.join(__dirname, 'eval-nodejs-handler')),
       });
+    case lambda.Runtime.NODEJS_12_X:
```

Comment:
❗  The case statement above means these sections will never run.

Reply:
`goto fail`. Thanks!